.php-cs-fixer.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+use Ibexa\CodeStyle\PhpCsFixer\InternalConfigFactory;
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
+    ->files()->name('*.php');
```

Comment:
I know that internally implementation will exclude `vendor` directory  but I would really prefer to explicit list `src` and `tests` directories.